applications/jonny5/src/j5_limits.erl

```diff
@@ -64,10 +67,12 @@
 -spec get(ne_binary()) -> limits().
 get(Account) ->
     AccountId = wh_util:format_account_id(Account, 'raw'),
-    case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
-        {'ok', Limits} -> Limits;
+    Limits = case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
+        {'ok', _Limits} -> _Limits;
```

Comment:
I made a change just to the var name as you suggested (plus copyright and
contributors) but now for some reason github thinks the whole limits file
has changed.
Trying to work that out.

On Mon, Feb 2, 2015 at 11:14 AM, James Aimonetti <notifications@github.com>
wrote:

> In applications/jonny5/src/j5_limits.erl
> <https://github.com/2600hz/kazoo/pull/735#discussion_r23949432>:
>
> > @@ -64,10 +67,12 @@
> >  -spec get(ne_binary()) -> limits().
> >  get(Account) ->
> >      AccountId = wh_util:format_account_id(Account, 'raw'),
> > -    case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
> > -        {'ok', Limits} -> Limits;
> > +    Limits = case wh_cache:peek_local(?JONNY5_CACHE, ?LIMITS_KEY(AccountId)) of
> > +        {'ok', _Limits} -> _Limits;
>
> Using the underscore-prefixed variable name is bad practice (other than
> when logging). Maybe just use {'ok', Lims} -> Lims; or something
> similarly ghetto :). Using _Limits reads as "This Variable contains the
> limits but I don't care about it here and it should be ignored", which
> isn't the case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/2600hz/kazoo/pull/735/files#r23949432>.
>